src/config/api.ts

```diff
@@ -37,8 +37,8 @@ api.interceptors.response.use(
         originalRequest.headers.Authorization = `Bearer ${accessToken}`;
         return axios(originalRequest);
       } catch (error) {
-        // todo: proper error handling
-        alert('Could not refresh token');
+        // todo: check what does it do
```

Comment:
Is this supposed to be here?

Reply:
kind of. it is hard to do any error handling in an interceptor so the only possible solution is to refresh the page and hope it does not reproduce on the refresh. downside is that if it reproduces, we will be stuck in a refresh loop.